src/test/java/uk/gov/pay/api/it/validation/PaymentsResourceEmailValidationITest.java

```diff
@@ -0,0 +1,40 @@
+package uk.gov.pay.api.it.validation;
+
+import org.junit.Before;
+import org.junit.Test;
+import uk.gov.pay.api.it.PaymentResourceITestBase;
+import uk.gov.pay.api.utils.PublicAuthMockClient;
+
+import java.util.Map;
+
+import static io.restassured.http.ContentType.JSON;
+import static org.hamcrest.core.Is.is;
+
+public class PaymentsResourceEmailValidationITest extends PaymentResourceITestBase {
+
+    private PublicAuthMockClient publicAuthMockClient = new PublicAuthMockClient(publicAuthMock);
+
+    @Before
+    public void setUpBearerToken() {
+        publicAuthMockClient.mapBearerTokenToAccountId(API_KEY, GATEWAY_ACCOUNT_ID);
+    }
+    
+    @Test
+    public void shouldRespondWith422_whenEmailIsGreaterThan254Chars() {
+
+        String payload = toJson(
+                Map.of("amount", 100, 
+                        "reference", "Some ref", 
+                        "description", 
+                        "hi", "return_url", "https://somewhere.gov.uk/rainbow/1",
+                       "email", "aVeryLongEaVeryLongEaVeryLongEaVeryLongEaVeryLongEaVeryLongEaVeryLongEaVeryLongEaVeryLongEaVeryLongEaVeryLongEaVeryLongEaVeryLongEaVeryLongEaVeryLongEaVeryLongEaVeryLongEaVeryLongEaVeryLongEaVeryLongE@aVeryLongEaVeryLongEaVeryLongEaVeryLongEaVeryLongEaVeryLongEaVeryLongEaVeryLongEaVeryLongEaVeryLongE.com"));
```

Comment:
In Java 11 we can do something like `" ".repeat(255);` which I think looks a bit nicer.